internal/jobcontainers/jobcontainer.go

```diff
@@ -181,24 +235,40 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return name
 	}
 
-	workDir := c.sandboxMount
-	if conf.WorkingDirectory != "" {
-		var changed bool
-		// For now, we join the working directory requested with where the sandbox volume is located. It's expected that the default behavior
-		// would be to treat all paths as relative to the volume.
-		//
-		// For example:
-		// A working directory of C:\ would become C:\C\12345678\
-		// A working directory of C:\work\dir would become C:\C\12345678\work\dir
-		//
-		// The below calls replaceWithMountPoint to replace any occurrences of the environment variable that points to where the container image
-		// volume is mounted.
-		workDir, changed = c.replaceWithMountPoint(conf.WorkingDirectory)
-		// If the working directory was changed, that means the user supplied %CONTAINER_SANDBOX_MOUNT_POINT%\\my\dir or something similar.
-		// In that case there's nothing left to do, as we don't want to join it with the mount point again.. If it *wasn't* changed, then we
-		// need to join it with the mount point, as it's some normal path.
-		if !changed {
-			workDir = filepath.Join(c.sandboxMount, removeDriveLetter(workDir))
+	workDir := c.rootfsLocation
+	// Only need to adjust the working directory in the case where bind support isn't available.
+	if !fileBindingSupport {
+		if conf.WorkingDirectory != "" {
+			var changed bool
+			// For now, we join the working directory requested with where the sandbox volume is located. It's expected that the default behavior
+			// would be to treat all paths as relative to the volume.
+			//
+			// For example:
+			// A working directory of C:\ would become C:\C\12345678\
+			// A working directory of C:\work\dir would become C:\C\12345678\work\dir
+			//
+			// The below calls replaceWithMountPoint to replace any occurrences of the environment variable that points to where the container image
+			// volume is mounted.
+			workDir, changed = c.replaceWithMountPoint(conf.WorkingDirectory)
+			// If the working directory was changed, that means the user supplied %CONTAINER_SANDBOX_MOUNT_POINT%\\my\dir or something similar.
+			// In that case there's nothing left to do, as we don't want to join it with the mount point again.. If it *wasn't* changed, then we
+			// need to join it with the mount point, as it's some normal path.
+			if !changed {
+				workDir = filepath.Join(c.rootfsLocation, removeDriveLetter(workDir))
+			}
+		}
+	} else {
+		// Bind support is available, set the default working directory to C:\payload if
```

Comment:
C:\payload

Reply:
This comment is honestly kinda misplaced anyways, changing the text.

Reply:
Updated!